src/main/java/com/commercetools/sync/services/TypeService.java

```diff
@@ -26,4 +31,35 @@
      */
     @Nonnull
     CompletionStage<Optional<String>> fetchCachedTypeId(@Nonnull final String key);
+
```

Comment:
Please cover the newly added services with integration tests.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/304/commits/2dc8da912803fd607ac617649665927fa749e81d